### ISSUE TITLE
Soundposts using a .webm as the audio source don't play any audio.

### ISSUE THREAD

**Issue by Kurogal** [K1rakishou/Kuroba-Experimental#1071]
It seems to effect all webms even if it's just an opus audio stream with no video.
https://archive.palanq.win/vt/thread/74848317
I posted all the ones in that thread (minus the mp3) and all but the first one using opus is just the webm audio straight from yt-dlp.
I quickly did the opus sound post from my phone before that using Seal which seems to automatically convert the webm to opus if I just download the audio, and as far as I'm aware you can copy an opus audio stream straight from the .webm to an .opus through ffmpeg without re-encoding it.

I also tend to see webms that are just the original video and audio together being used as the audio.
https://archive.palanq.win/vt/thread/74410289/#q74438782
Searching for webm] shows all of the ones using webm audio, static and video.
I linked one specifically using both a webm video and a webm with video and audio for convenience in case it's relevant for testing, but pretty much all of the ones I checked using both have that as the case in the thread.
The audio player also won't appear for webm soundposts using a static image which makes me assume it is specifically webms as an audio source that are the issue.
I don't have any problems with any other soundposts in either thread with KurobaEx.
No problems with the same webm sound posts on desktop either using [4chan sounds player](https://github.com/vampiricwulf/4chan-sounds-player) even in the archived threads. I added RozenArcana to the script to test the webms and it worked fine still.

I'm not sure there's something obvious I'm missing and/or this is hard to fix, but I couldn't find any other issues mentioning this specifically.
Also it doesn't seem like quotes work in app when browsing RozenArcana, no quoted by and clicking on a quote takes me to the browser rather than the post in the app. I thought I'd mention since I didn't see an issue on it yet either.

**Comment by K1rakishou** [comment]
Yeah it was never implemented because the implementation wasn't easy to do:
https://github.com/K1rakishou/Kuroba-Experimental/blob/d8205405f17b023baa95ccce6d67abd69327d4e2/Kuroba/app/src/main/java/com/github/k1rakishou/chan/features/media_viewer/ViewableMediaParcelableHolder.kt#L404

**Comment by K1rakishou** [comment]
This is finally fixed.
